### PR TITLE
Change transaction hash in payload from string to Uint256.

### PR DESCRIPTION
**Pull request by RainFallsSilent**

### CODE REVIEW

File: blockchain/txpool_test.go [hunk 1]
```diff
@@ -46,15 +46,25 @@ func TestTxPoolInit(t *testing.T) {
...
 func TestTxPool_VerifyDuplicateSidechainTx(t *testing.T) {
+	hashStr1 := "8a6cb4b5ff1a4f8368c6513a536c663381e3fdeff738e9b437bd8fce3fb30b62"
+	hashBytes1, _ := common.HexStringToBytes(hashStr1)
+	hash1, _ := common.Uint256FromBytes(hashBytes1)
+	hashStr2 := "cc62e14f5f9526b7f4ff9d34dcd0643dacb7886707c57f49ec97b95ec5c4edac"
+	hashBytes2, _ := common.HexStringToBytes(hashStr2)
+	hash2, _ := common.Uint256FromBytes(hashBytes2)
+	hashStr3 := "8a6cb4b5ff1a4f8368c6513a536c663381e3fdeff738e9b437bd8fce3fb30b62"
+	hashBytes3, _ := common.HexStringToBytes(hashStr3)
+	hash3, _ := common.Uint256FromBytes(hashBytes3)
```
Comment (yz89) [inline blockchain/txpool_test.go]: hash3 is same with hash1

File: blockchain/txpool_test.go [hunk 2]
```diff
@@ -73,8 +83,8 @@ func TestTxPool_VerifyDuplicateSidechainTx(t *testing.T) {
 	txn2.Payload = &core.PayloadWithdrawFromSideChain{
 		BlockHeight:         100,
 		GenesisBlockAddress: "eb7adb1fea0dd6185b09a43bdcd4924bb22bff7151f0b1b4e08699840ab1384b",
-		SideChainTransactionHashes: []string{
-			"8a6cb4b5ff1a4f8368c6513a536c663381e3fdeff738e9b437bd8fce3fb30b62", // duplicate sidechain Tx
+		SideChainTransactionHashes: []common.Uint256{
+			*hash3, // duplicate sidechain Tx
```
Comment (yz89) [inline blockchain/txpool_test.go]: should use hash1

File: blockchain/txvalidator_test.go [hunk 3]
```diff
@@ -330,16 +330,26 @@ func TestCheckTransactionPayload(t *testing.T) {
...
 		BlockHeight:         100,
 		GenesisBlockAddress: "eb7adb1fea0dd6185b09a43bdcd4924bb22bff7151f0b1b4e08699840ab1384b",
-		SideChainTransactionHashes: []string{
-			"8a6cb4b5ff1a4f8368c6513a536c663381e3fdeff738e9b437bd8fce3fb30b62",
-			"cc62e14f5f9526b7f4ff9d34dcd0643dacb7886707c57f49ec97b95ec5c4edac",
-			"8a6cb4b5ff1a4f8368c6513a536c663381e3fdeff738e9b437bd8fce3fb30b62", // duplicate tx hash
+		SideChainTransactionHashes: []common.Uint256{
+			*hash1,
+			*hash2,
+			*hash3, // duplicate tx hash
```
Comment (yz89) [inline blockchain/txvalidator_test.go]: use hash1 here

File: blockchain/txpool_test.go (hunk 1 above)
Comment (RainFallsSilent) [reply blockchain/txpool_test.go]: has fixed need review again

File: blockchain/txvalidator_test.go (hunk 3 above)
Comment (RainFallsSilent) [reply blockchain/txvalidator_test.go]: has fixed need preview again

File: blockchain/txpool_test.go (hunk 2 above)
Comment (RainFallsSilent) [reply blockchain/txpool_test.go]: has fixed need review again

**Pull request merged**